imagersite/imager_images/tests.py

```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
+
+# Create your tests here.
```

Comment:
Woah, these rogue models are out and about without any tests reigning them in!  This is dangerous!